src/Glpi/Asset/Capacity/HasOperatingSystemCapacity.php

```diff
@@ -52,6 +53,12 @@ public function getIcon(): string
         return OperatingSystem::getIcon();
     }
 
+    #[Override]
+    public function getDescription(): string
+    {
+        return __("Display operating system informations found by automatic inventory");
```

Comment:
Not only from automatic inventory

Reply:
Also, `information` is uncountable
```suggestion
        return __("Display operating system information");
```